### PR TITLE
Add old to URL table map to 404 page

### PR DESCRIPTION
**Pull request by sravfeyn**
This will make it easy for people to find the new link

@joxl 

Screenshot 2022-07-26 at 12 48 31 PM

### CODE REVIEW

File: docs/404.md [hunk 1]
```diff
@@ -2,4 +2,75 @@
...
+|                                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                  |
+| [https://dimagi.github.io/commcare-cloud/firefighting/index](https://dimagi.github.io/commcare-cloud/firefighting/index)                                                                        | [https://commcare-cloud.readthedocs.io/en/latest/reference/firefighting/index.html](https://commcare-cloud.readthedocs.io/en/latest/reference/firefighting/index.html)                                                                                                           |
+                                        |
```
Comment (joxl) [inline docs/404.md:29]: Is this row malformed? When I view the rendered version of this file the rows below here aren't in a table format.

**Review by joxl — COMMENTED** [review]
Thanks for this @sravfeyn!

File: docs/404.md (hunk 1 above)
Comment (sravfeyn) [reply docs/404.md:29]: I have just fixed it.

**Comment by sravfeyn** [comment]
I haven't explored too deeply, but if the github pages allows a site-wide javascript snippet, we could query this table for the current URL and auto redirect to the new pages without user having to do anything!

**Comment by joxl** [comment]
> we could query this table for the current URL and auto redirect to ...

That would be fancy! Unless it takes only very little effort, I'm inclined to say we it's probably not _that_ important yet. Definitely good to keep on the table as something to explore if needed, though.

**Review by joxl — APPROVED** [review]

**Pull request merged**